mopidy/core/actor.py

```diff
@@ -58,19 +58,27 @@ def __init__(self, mixer=None, backends=None):
         self.tracklist = TracklistController(core=self)
 
     def get_uri_schemes(self):
+        """Get list of URI schemes we can handle"""
         futures = [b.uri_schemes for b in self.backends]
         results = pykka.get_all(futures)
         uri_schemes = itertools.chain(*results)
         return sorted(uri_schemes)
 
     uri_schemes = property(get_uri_schemes)
```

Comment:
Perhaps we could do `property(deprecated(get_uri_schemes, message='....'))` via a decorator and use `warnings` to start logging uses of this (possibly just as a pending deprecation)?